pkg/receive/handler.go

```diff
@@ -629,6 +599,77 @@ func isNotReady(err error) bool {
 		status.Code(err) == codes.Unavailable
 }
 
+// countCause counts the number of errors within the given error slice
+// whose causes satisfy the given function.
+// cause will inspect the error's cause or, if the error is a MultiError,
```

Comment:
is it important? 

because you know if we use `someunkownimportpackage.YoloError` it will be not traversed either ;p